src/index.html

```diff
@@ -5,9 +5,317 @@
+                name="name"
+                placeholder="name">
+            </label>
+
+            <label for="phone">
+              <input
+                class="input form__input"
+                id="phone"
+                type="tel" name="phone"
+                placeholder="phone">
+            </label>
+
+            <label for="email">
+              <input
+                class="input form__input"
+                id="email"
+                type="email" name="email"
+                placeholder="email">
+            </label>
+
+            <button class="button form__button">
+              Consultation
+            </button>
+          </form>
+        </div>
+      </section>
+
+      <section class="section">
+        <h2 class="heading section__heading">
+          Testimonials<span class="heading__crown"></span>
+        </h2>
+        <div class="testimonials section__content">
+          <div class="person">
+            <img
+              src="images/people/1.jpg"
+              alt="Tim Cook"
+              class="person__photo">
+            <h3 class="person__name">
+              Tim Cook
+            </h3>
+            <p class="person__position">
+              CEO
+            </p>
+            <p class="person__text">
+              Nice work, Certificates National Association of Realtors (USA).
+            </p>
+          </div>
+
+          <div class="person">
+            <img
+              src="images/people/2.jpg"
+              alt="Katherine Adams"
+              class="person__photo">
+            <h3 class="person__name">
+              Katherine Adams
+            </h3>
+            <p class="person__position">
+              Senior Vice President
+            </p>
+            <p class="person__text">
+              My soul is illuminated by an unearthly joy,
+              as these beautiful spring morning, which I enjoy with all my heart.
+              I’m all alone and blissfully happy in the local edge.
+            </p>
+          </div>
+
+          <div class="person">
+            <img
+              src="images/people/3.jpg"
+              alt="Jonathan Ive"
+              class="person__photo">
+            <h3 class="person__name">
+              Jonathan Ive
+            </h3>
+            <p class="person__position">
+              Chief Design Officer
+            </p>
+            <p class="person__text">
+              When my lovely valley of steam rises and half-day sun is above an impermeable.
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="section">
+        <h2 class="heading section__heading">
+          Contact us <span class="heading__crown"></span>
+        </h2>
+        <div class="contact">
+          <div class="contact__text">
+            <p>
+              <a href="mailto:Hello@miami.com" class="link contact__link">
+                Hello@miami.com
+              </a>
+              <br>
+                +1 234 555-55-55
```

Comment:
Це можна зробити як ссилку з параметром href-y "tel:..."
```suggestion
                <a href"tel:..." class="...">+1 234 555-55-55</a>
```